controllers/discovery/initDiscoveryTopicData.js

```diff
@@ -1,5 +1,6 @@
 const {QueryTypes} = require('sequelize');
-const {sequelize} = require('../../databases/models');
+const {User, sequelize} = require('../../databases/models');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `User` and `UsersFunction` is new in this hunk. Please ensure that these modules are correctly exported from their respective files, and that they are used appropriately within this file.
```diff
- const {sequelize} = require('../../databases/models');
+ const {User, sequelize} = require('../../databases/models');
+ const UsersFunction = require('../../databases/functions/users');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->